packages/server/src/api/routes/row.ts

```diff
@@ -177,7 +175,7 @@ router
     "/api/:tableId/rows",
```

Comment:
Can this tableId be the viewId from now on? If so, should we call it differently? SourceId or similar? I can see this causing confusion and future bugs